src/resolvers/Authentication.ts

```diff
@@ -2,10 +2,10 @@ import Authentications from "../services/Authentications";
 
 const resolvers = {
   Mutation: {
-    Login: ({ email, password }) =>
-      Authentications.login({ email, password }),
-    Register: ({ email, password }) =>
-      Authentications.register({ email, password }),
+    Login: ({ email, password, res }) =>
```

Comment:
Dont think this works. https://www.apollographql.com/docs/apollo-server/data/resolvers/

The first argument to the resolver is parent. Parent should not have email, password fields. You will need to say:
```js
Login: (_, {email, password}, {response})
```

<img width="730" alt="image" src="https://github.com/include-davis/Include-Estore-API/assets/66186160/2afaa127-7621-43b7-9a53-14c11c53f504">

Please test your code first before requesting review.